galley/pkg/config/analysis/analyzers/util/conversions.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package util
+
+import (
+	"istio.io/api/networking/v1alpha3"
+	"istio.io/istio/galley/pkg/config/analysis"
+	"istio.io/istio/galley/pkg/config/meta/metadata"
+	"istio.io/istio/galley/pkg/config/resource"
+)
+
+// ServiceEntriesToFqdnMap transforms IstioNetworkingV1Alpha3Serviceentries hostnames
+// to a map with ScopedFqdn key
+func ServiceEntriesToFqdnMap(ctx analysis.Context) map[ScopedFqdn]bool {
```

Comment:
This name is slightly confusing: it implies the return value is a map of service entry -> fqdn.

In https://github.com/istio/istio/pull/18674 I ended up calling this `initServiceEntryHostMap`, which I mostly like but suspect could be better. Perhaps once we've combined them we could call it `GetFqdnToServiceEntry`?